src/etl/sds/trigger/bulk/steps.py

```diff
@@ -42,49 +39,56 @@ def _validate_database_is_empty(data, cache: Cache):
 
 def _create_state_machine_input(data, cache):
     _, source_key = data[StepChain.INIT]
-    return StateMachineInput.bulk(changelog_number=Path(source_key).stem)
+    state_machine_input = StateMachineInput.bulk(changelog_number=Path(source_key).stem)
+    return state_machine_input
 
 
-def _copy_to_state_machine(data, cache: Cache):
-    source_bucket, source_key = data[StepChain.INIT]
-    return cache["s3_client"].copy_object(
-        Bucket=source_bucket,
-        Key=WorkerKey.EXTRACT,
-        CopySource=f"{source_bucket}/{source_key}",
-    )
-
-
-def _copy_to_history(data, cache: Cache):
+def _copy_to_intermediate_history_file(data, cache: Cache):
     source_bucket, source_key = data[StepChain.INIT]
     state_machine_input: StateMachineInput = data[_create_state_machine_input]
     return cache["s3_client"].copy_object(
         Bucket=source_bucket,
-        Key=f"history/{state_machine_input.etl_type}/{source_key}",
+        Key=f"history/{state_machine_input.name}",
         CopySource=f"{source_bucket}/{source_key}",
     )
 
 
-def _delete_object(data, cache: Cache):
+def _delete_trigger_object(data, cache: Cache):
     source_bucket, source_key = data[StepChain.INIT]
     return cache["s3_client"].delete_object(Bucket=source_bucket, Key=source_key)
 
 
-def _start_execution(data, cache):
+def _publish_message_to_sqs_queue(data, cache: Cache):
+    sqs_client = boto3.client("sqs")
```

Comment:
best practice is to declare boto3 clients at the handler level of the lamdba and cache them (e.g. with `s3_client`), although in this case (because bulk is only run once) this is defo v pedantic!